README.md

```diff
@@ -92,6 +122,84 @@ To enable TypeScript support you'll need to setup [`svelte-preprocess`](https://
    npm install typescript svelte-preprocess ts-jest -D
    ```
 
+#### ESM version
+
+1. Create a `svelte.config.js` at the root of your project:
+
+   ```js
```

Comment:
hi @sebastianrothe this section is broken, it doesn't show the file content

Reply:
thanks @rallets. I just pushed a fix for that